about/services/overview.md

```diff
@@ -0,0 +1,77 @@
+# Overview of hub services
+
+2i2c builds and operates **distributions of JupyterHubs** that are tailored for particular use-cases.
+These services share many of the same infrastructure components, but have customizations and optimizations that are more domain- or community-specific.
+
+:::{note}
+Our services are in an "alpha" state - we are still learning a lot about the best way that these hubs can serve communities in research and education.
+The infrastructure and service may change over the coming months!
+See [our strategy page](../strategy.md) for an overview of what we're hoping to do and where we're headed next.
+:::
+
+## What technology makes up each hub?
+
+🚀 core infrastructure
+: Underneath each 2i2c JupyterHub is a [JupyterHub](https://jupyter.org/hub). These provide interactive computing sessions for each of your users, and connect to the other infrastructure in the cloud. We use [`auth0`](https://auth0.com/) for authenticating users, which can connect to a number of other authentication protocols (such as OAuth2).
```

Comment:
".....use [`auth0`](https://auth0.com/) and [CILogon](https://www.cilogon.org/) for authenticating...."